docs/developing/auth.md

```diff
@@ -25,7 +25,7 @@ requests. By default, it passes requests to your app via a tool called
 ### Headers that an app receives
 
 Per the
-[current implementation](https://github.com/sandstorm-io/sandstorm/blob/411b344f3acb151693036f3c061b153a2fd91d68/src/sandstorm/sandstorm-http-bridge.c%2B%2B)
+[current implementation](https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/sandstorm-http-bridge.c%2B%2B)
```

Comment:
+1 to your change

It is somewhat laughable to link to an old commit and call it the "current implementation" :)